### PR TITLE
Postman Sort in C++

### LINKED ISSUE #2251: Postman sort
- [ ] C - @jhasaurabh312
- [x] C++ - @coderprasukj
- [x] C#
- [ ] CoffeeScript
- [ ] Dart - @Rukmini-Meda
- [ ] Go
- [ ] Java - @bluejay229
- [x] JS - @coderprasukj
- [ ] Kotlin - @myselfanuj
- [ ] Php
- [ ] Python - @jhasaurabh312
- [x] Readme - @shwetagurnani
- [ ] Ruby
- [ ] Rust
- [ ] TypeScript

### PR DESCRIPTION
**Pull request by coderprasukj**
Fixes #2251 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Added Postman Sort in C++

#### Languages Used:
- C++

#### Files Added:
- Postman_Sort.cpp

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by coderprasukj** [comment]
@jainaman224 @somya-kapoor @singh-shreya6 I have requested to view my pull request. Please go throught it.

File: Postman_Sort/Postman_Sort.cpp [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+int t1, t2, k, t, n = 1, i, j;
+int arr[100], arr1[100];
+int main(){
+    int count, max;
+    cout << "The size of the Array : " ;
+    cin >> count;
+    int arr[count];
+    for(i = 0; i < count; i++)
+        cin >> arr[i];
+    cout << "\nThe Unsorted Array is : \n" ;
```
Comment (singh-shreya6) [inline Postman_Sort/Postman_Sort.cpp:17]: Please leave a blank line before this.

**Comment by singh-shreya6** [comment]
Post making your changes. Squash your commits too.

**Comment by coderprasukj** [comment]
@singh-shreya6 I have done the required changes and have squashed the commits. Kindly have a look at it.

**Pull request merged**